test/test_utils/ec2.py

```diff
@@ -13,18 +13,20 @@
 from invoke import run
 from packaging.version import Version
 from packaging.specifiers import SpecifierSet
-from tenacity import retry, stop_after_attempt, wait_fixed
+from tenacity import (
+    retry, stop_after_attempt, stop_after_delay, wait_fixed, wait_random_exponential
+)
```

Comment:
Formatting check failed here. Will fix in next revision of the PR alongside any other changes requested.